soc/arm/nuvoton_npcx/common/soc_clock.h

```diff
@@ -0,0 +1,28 @@
+ /*
+ * Copyright (c) 2020 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _NUVOTON_NPCX_SOC_CLOCK_H_
+#define _NUVOTON_NPCX_SOC_CLOCK_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Common clock control device name for all NPCX series */
+#define NPCX_CLOCK_CONTROL_NAME "npcx-cc"
+
+/* Structure for clock controller config */
+struct npcx_clk_cfg {
+	uint16_t bus     : 8;
```

Comment:
I'm not sure about style in Zephyr, so please ignore this if you know better, but I am more used to:

bus:8;
ctrl:5;

(i.e. no spaces)

A quick look at the code and I don't see spaces before bitfield sizes, but I might be missing something